### DISCUSSION TITLE
Door Annotation corners model train

### DISCUSSION THREAD

**Discussion by omoigwa93** [chartjs/chartjs-plugin-annotation#767] [Q&A]
Can someone help on how to setup/train model-door corners annotations(right corner, Left corner, Bottom left corner and bottom right corner) and some categorization. Please🙏

**Comment by stockiNail** [comment]
@omoigwa93 I have to admit that I don't know exactly what `train model-door` means but I assume we are talking about rectangles (or better boxes) with rounded corners. 
Box and label annotations accept [borderRadius](https://www.chartjs.org/chartjs-plugin-annotation/latest/guide/types/box.html#styling) option where you can set the rounded corner.

But if you need more complex shape to draw, I think the best option is to define a label annotation where the content is a canvas (or image) where you have the freedom to draw whatever you want.

If you can provide here an example about a train model-door shape you would like to have, I can try to help to design it.
